src/main/java/Main.java

```diff
@@ -62,4 +76,24 @@ private void wellcome(String name, String account) {
         print("Your current account is " + account);
     }
 
+//    public char readFirstChar() {
+//        return (char) scanner.nextLine().trim().toUpperCase().charAt(0);
+//    }
+
+    private void withdraw(BankAccount bankAccount) {
+        double retiro;
+        Scanner scanner = new Scanner(System.in);
```

Comment:
**Resource leak** "scanner" is never closed.

Try to open just one Scanner object and send it as a parameter when you need it